coursier.bzl

```diff
@@ -291,7 +301,9 @@ def _pinned_coursier_fetch_impl(repository_ctx):
                 http_files.append("        urls = [\"%s\"]," % artifact["url"])
             http_files.append("    )")
     repository_ctx.file("defs.bzl", "\n".join(http_files), executable = False)
-    repository_ctx.file("netrc", "\n".join(get_netrc_lines_from_entries(netrc_entries)), executable = False)
+    repository_ctx.file("netrc", "\n".join(
+        get_home_netrc_contents(repository_ctx).splitlines() + get_netrc_lines_from_entries(netrc_entries)),
```

Comment:
can you add a comment that netrc entries are order-dependent, and we respect the HOME netrc contents over the inlined auth info?